charts/nextcloud/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: nextcloud
-version: 2.9.1
-appVersion: 21.0.4
+version: 2.10.0
+appVersion: 21.0.7
```

Comment:
I am not seeing a 21.0.7 docker image on dockerhub

Reply:
@mrueg

Reply:
@tvories yes that was a bit too early, I just saw the release 21.0.7 on nextcloud/server happening. I believe it will be pushed in the next couple of hours.

Reply:
Alternatively I can bump to nc-22?

Reply:
I was thinking go to 22 since that's the latest.

Reply:
Okay, I bumped it to 22.2.3!